src/Image.cc

```diff
@@ -182,7 +184,7 @@ NAN_SETTER(Image::SetHeight) {
  * Get src path.
  */
 
-NAN_GETTER(Image::GetSource) {
+void Image::GetSource(const Nan::FunctionCallbackInfo<v8::Value> &info) {
```

Comment:
I think this and line 227 should/could be `NAN_METHOD(Image::GetSource)` ?

Reply:
Rebased properly?